ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -326,22 +326,25 @@ objects:
           echo $BOOTSTRAP_FIP
         }
 
-
-
-
-        function create_api_fip () {
-          #(TODO) adduarte: all openstack resources should be written toa file which can be used to delete later.
-          LB_FIP_AND_ID=$(openstack floating ip create --description api.$CLUSTER_NAME.$BASE_DOMAIN $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address' -c 'id')
+        function create_api_fip() {
+          LB_FIP_AND_ID=$(openstack floating ip create --description "api.$CLUSTER_NAME.$BASE_DOMAIN" "$OPENSTACK_EXTERNAL_NETWORK" --format value -c 'floating_ip_address' -c 'id')
           LB_FIP_IP=$(echo $LB_FIP_AND_ID |awk '{print $1}')
           LB_FIP_UID=$(echo $LB_FIP_AND_ID |awk '{print $2}')
-          echo $LB_FIP_UID > ${TMP_SHARED}/LB_FIP_UID
+          echo $LB_FIP_UID > "${TMP_SHARED}/LB_FIP_UID"
         }
 
         function create_ingress_fip() {
-          INGRESS_FIP_AND_ID=$(openstack floating ip create --description ingress.$CLUSTER_NAME.$BASE_DOMAIN $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address' -c 'id')
+          INGRESS_FIP_AND_ID=$(openstack floating ip create --description "ingress.$CLUSTER_NAME.$BASE_DOMAIN" "$OPENSTACK_EXTERNAL_NETWORK" --format value -c 'floating_ip_address' -c 'id')
           INGRESS_FIP_IP=$(echo $INGRESS_FIP_AND_ID |awk '{print $1}')
           INGRESS_FIP_UID=$(echo $INGRESS_FIP_AND_ID |awk '{print $2}')
-          echo $INGRESS_FIP_UID > ${TMP_SHARED}/INGRESS_FIP_UID
+          echo $INGRESS_FIP_UID > "${TMP_SHARED}/INGRESS_FIP_UID"
+        }
+
+        function create_bootstrap_fip() {
+          FIP_ID=$(openstack floating ip create --description "bootstrap.$CLUSTER_NAME.$BASE_DOMAIN" "$OPENSTACK_EXTERNAL_NETWORK" --format value -c 'id')
+          FIP_IP=$(openstack floating ip show "$FIP_ID" --format value -c 'floating_ip_address')
+          echo $FIP_ID > "${TMP_SHARED}/BOOTSTRAP_FIP_UID"
+          printf '%s' "$FIP_IP"
```

Comment:
what for?  This will appear just as an ip address somewhere in the logs?  maybe add a descriptor or just remove.